WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -173,7 +179,8 @@
           <Border Classes="warning info">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction uses coinjoining coins." />
+              <TextBlock Text="Transaction uses coinjoining coins." 
+                         ToolTip.Tip="This may cause issues. For example, your transaction might be replaced by the coinjoin one. Consider waiting for the current coinjoin to be finished before sending this transaction." />
```

Comment:
Tbh the user shouldn't care about the coinjoin that is happening, and we should make sure that we do not replace users txs with coinjoin by not allowing spent coins to sign coinjoin txs.

Once the user waits for the coinjoin the coins' anonymity score might be lower than before and he will end up with unconfirmed coins, so now he has to wait more?